spec/migrations/20220223095704_remove_col3_from_miq_widget_set_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_migration
+
+describe RemoveCol3FromMiqWidgetSet do
+  let(:widget_stub) { migration_stub(:MiqWidgetSet) }
+
+  migration_context :up do
+    it "Moving col3 widgets to col2 and removing col3" do
+      data = {:col1 => [1, 2, 3], :col2 => [4, 5, 6], :col3 => [7, 8, 9]}
+      set_data = {:col1 => data[:col1], :col2 => (data[:col2] + data[:col3]).uniq.compact}
+      widget = widget_stub.create(:set_data => set_data)
+      migrate
+      expect(widget.reload.set_data).to eq({:col1 => [1, 2, 3], :col2 => [4, 5, 6, 7, 8, 9]}.to_s)
```

Comment:
And then when you add the serialize, this can just be a normal Hash comparison (i.e. you can remove the .to_s)